pygef/broxml.py

```diff
@@ -52,128 +51,79 @@ def __init__(self, path=None, string=None):
         super().__init__(path=path, string=string)
         self.nen_version = "NEN-EN-ISO 14688"
         self.type = "bore"
+        self._root = etree.fromstring(self.s)
+        self._ns_map_keys = list(self._root.nsmap.keys())
+
+        ns_map_values = [
```

Comment:
I think it's better to make this a global constant than a local variable.

Reply:
agree